company-quickhelp.el

```diff
@@ -184,7 +184,8 @@ currently active `company' completion candidate."
              (overlay-width (* (frame-char-width)
                                (if ovl (overlay-get ovl 'company-width) 0)))
              (overlay-position (* (frame-char-width)
-                                  (- (if ovl (overlay-get ovl 'company-column) 1) 1)))
+                                  (+ (- (if ovl (overlay-get ovl 'company-column) 1) 1)
+                                     (line-number-display-width) 2)))
```

Comment:
Does with work well with line numbers disabled as well?

And in versions of Emacs that don't have this function?

You can take `company--posn-col-row` as an example for fixing this.